src/TIMEZONES.ts

```diff
@@ -419,3 +420,133 @@ export default [
     'Pacific/Wake',
     'Pacific/Wallis',
 ] as const;
+
+const timezoneBackwardMap: Record<string, (typeof TIMEZONES)[number]> = {
```

Comment:
Could you add a comment explaining briefly where these values come from (link to wikipedia page)/what the purpose of the map is? That way people curious in the future don't have to follow the GH blame trail.